.gitignore

```diff
@@ -388,3 +388,6 @@ MigrationBackup/
 
 # Output from the "download_spec_sources.sh" script
 downloaded_sources/
+
+# build_container from the containerized-build tool
+toolkit/scripts/containerized-build/build_container/
```

Comment:
Is there a particular reason why this folder multiple subdirs into the code hierarchy is being gitignored?

Reply:
This is a temp folder of sorts, where the tool stores
- hash of the worker_chroot so that a new image is created only when the worked_chroot has changed.
- tools and pkg-graphs from Mariner toolkit so they can copied into the container
Changes to this folder should not be added to git

Reply:
Can the temp folder not be part of the primary toolkit scripts area? Typically temp collateral is part of a generated top-level `build` folder. Keeping all temp collateral in one area makes it easier to triage issues later, instead of having temp data interspersed in the repo.